bin/bootstrap

```diff
@@ -81,26 +82,27 @@ fi
 #
 # Github ssh key setup
 #
-stat ~/.ssh/github
+stat ~/.ssh/id_ed25519
```

Comment:
Would it better to still label this for use with github? 

```suggestion
stat ~/.ssh/github_ed25519
```

Reply:
Sure, renamed it